airbyte-integrations/connectors/destination-oracle/gradle.properties

```diff
@@ -0,0 +1 @@
+junitMethodExecutionTimeout = 30 m
```

Comment:
let's limit this to long methods rather than a blanket increase

Reply:
will do